optax/schedules/_schedule_test.py

```diff
@@ -654,6 +654,13 @@ def test_invalid_scale(self):
     with self.assertRaises(ValueError):
       _schedule.piecewise_interpolate_schedule('linear', 13.0, {5: -3})
 
+  def test_zero_interval_size(self):
+    sched = _schedule.piecewise_interpolate_schedule(
+      'linear', 13.0, {0: 1.0, 5: 1.0}
+    )
+    if not all(jnp.isfinite(sched(i)) for i in range(6)):
+      raise ValueError('Piecewise schedule gives nan values for finite scales.')
```

Comment:
I'm confused by this test, should this be an assertion? (Also it looks like the test deals with non-unique scales, not bounds?)

Reply:
The message is confusing and this should be an assert, thanks, fiixed!